lib/roo_on_rails/checks/environment.rb

```diff
@@ -4,13 +4,15 @@
 require 'roo_on_rails/checks/heroku/preboot_enabled'
 require 'roo_on_rails/checks/heroku/app_exists'
 require 'roo_on_rails/checks/sidekiq/settings'
+require 'roo_on_rails/checks/heroku/drains_metrics'
 
 module RooOnRails
   module Checks
     class Environment < EnvSpecific
       requires GitHub::BranchProtection
       requires Heroku::PrebootEnabled
       requires Sidekiq::Settings
+      requires Heroku::DrainsMetrics
```

Comment:
Being picky, would be nice to have these in alphabetical order (and maybe the `require`s too).